README.md

```diff
@@ -115,7 +148,30 @@ An example of extra specs json would look like this:
+    }
+```
+
+*NOTE*: The `extra_context` spec adds a map of key/value pairs that are expanded in the `runner_install_template`.
+The `runner_install_template` allows us to completely override the script that installs and starts the runner. In the example above, I have added a copy of the current template from `garm-provider-common`, with the adition of one line:
```

Comment:
Add a note that this is a [golang template](https://pkg.go.dev/text/template) which enables all the goodies that come with it. Give some examples if possible.

Reply:
Added a note that reference golang templates and their example regarding those